qemu/tests/seabios.py

```diff
@@ -60,55 +60,54 @@ def info_check():
         if not utils_misc.wait_for(info_check, timeout, 1):
             err_msg = "Cound not get sgabios message. The output"
             err_msg += " is %s" % get_output(vm.serial_console)
-            raise error.TestFail(err_msg)
+            raise test.fail(err_msg)
 
-    if restart_key:
-        error.context("Restart vm and check it's ok", logging.info)
+    if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
+        raise test.fail("Could not get boot menu message.")
 
-        if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
-            raise error.TestFail("Could not get boot menu message.")
+    if restart_key:
+        error_context.context("Restart vm and check it's ok", logging.info)
 
         seabios_text = get_output(seabios_session)
         headline = seabios_text.split("\n")[0] + "\n"
         headline_count = seabios_text.count(headline)
 
         vm.send_key(restart_key)
 
+        if not (boot_menu_hint and utils_misc.wait_for(boot_menu, timeout, 1)):
+            raise test.fail("Could not get boot menu message.")
```

Comment:
Should we make this prompt different from line 66, like `Could not get boot menu message after rebooting.` ?

Reply:
@sitoliu  good idea,  updated